gcp/modules/gke-cluster/main.tf

```diff
@@ -23,6 +23,12 @@ variable "node_count" {
   default = 1
 }
 
+# When this variable is blank, GKE-default K8s version is used.
+# Check data.gke_version_assert for more details.
+variable "kubernetes_version" {
+  default = "1.13.7-gke.8"
+}
+
 variable "expected_gke_version_prefix" {
   default = "1.12"
 }
```

Comment:
I find the interaction of `kubernetes_version` and `expected_gke_version_prefix` kind of confusing. Maybe change the latter to `expected_default_gke_version_prefix`?